libs/ktem/ktem/tags/pipelines.py

```
@@ -36,8 +37,9 @@ def tag_crud(self) -> TagCRUD:
 
     @classmethod
     def resolve_tag_names(cls, tag_str: str) -> list[str]:
+        tag_str = tag_str.strip()
         tags = []
-        if tag_str:
+        if tag_str != "":

```

we could fail fast in this case
```python
if tag_str.strip() == '':
    return []
return list(map(str.strip, tag_str.split(',')))
```